network/network.go

```diff
@@ -254,6 +237,18 @@ func NewNetwork(
 		return nil, fmt.Errorf("initializing network metrics failed with: %w", err)
 	}
 
+	ipTracker, err := newIPTracker(log)
+	if err != nil {
+		return nil, fmt.Errorf("initializing ip tracker failed with: %w", err)
+	}
+	config.Validators.RegisterCallbackListener(constants.PrimaryNetworkID, ipTracker)
+
+	// Track all default bootstrappers to ensure their current IPs are gossiped
+	// like validator IPs.
+	for _, bootstrapper := range genesis.GetBootstrappers(config.NetworkID) {
+		ipTracker.ManuallyTrack(bootstrapper.ID)
+	}
```

Comment:
Why don't we just rely on `Network#ManuallyTrack` for this instead? I thought the the caller in `node` package already did this for us by calling `ManuallyTrack` on the network instance

Reply:
`Network#ManuallyTrack` initiates connections to a selection of bootstrap nodes. This part of the code ensures that all of the default bootstrap nodes will have their most recent IPs gossiped.